pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -115,6 +117,67 @@ func InjectAutoInstrumentation(rawPod []byte, _ string, ns string, _ *authentica
 	return mutate(rawPod, ns, injectAutoInstrumentation, dc)
 }
 
+// LibIbjectionNamespaceLabelSelector returns a label selector with the namespaces where libraries should be injected
+func LibIbjectionNamespaceLabelSelector() *v1.LabelSelector {
+	if !isApmInstrumentationEnabled() {
+		return nil
+	}
+
+	apmEnabledNamespaces := config.Datadog.GetStringSlice("apm_config.instrumentation.enabled_namespaces")
+	apmDisabledNamespaces := config.Datadog.GetStringSlice("apm_config.instrumentation.disabled_namespaces")
+
+	// apm.instrumentation.enabled_namespaces and apm.instrumentation.disabled_namespaces configuration cannot be set at the same time
+	if len(apmEnabledNamespaces) > 0 && len(apmDisabledNamespaces) > 0 {
+		log.Error("apm.instrumentation.enabled_namespaces and apm.instrumentation.disabled_namespaces configuration cannot be set together")
+		return &v1.LabelSelector{
+			// Does not match anything because LabelMetadataName should always be set.
+			MatchExpressions: []v1.LabelSelectorRequirement{
+				{
+					Key:      corev1.LabelMetadataName,
+					Operator: v1.LabelSelectorOpDoesNotExist,
+				},
+			},
+		}
+	}
+
+	// "kube-system" and the datadog namespace are excluded by default, but can
+	// be included with apm.instrumentation.enabled_namespaces
+	if len(apmEnabledNamespaces) > 0 {
+		return &v1.LabelSelector{
+			MatchExpressions: []v1.LabelSelectorRequirement{
+				{
+					Key:      corev1.LabelMetadataName,
+					Operator: v1.LabelSelectorOpIn,
+					Values:   apmEnabledNamespaces,
+				},
+			},
+		}
+	}
+
+	labelSelector := v1.LabelSelector{
+		MatchExpressions: []v1.LabelSelectorRequirement{
+			{
+				Key:      corev1.LabelMetadataName,
+				Operator: v1.LabelSelectorOpNotIn,
+				Values:   []string{"kube-system", apiServerCommon.GetResourcesNamespace()},
```

Comment:
❓ 

I understand that the datadog namespace should be excluded by default. Where is this done? Is the datadog namespace obtained by `apiServerCommon.GetResourcesNamespace()` ?

Reply:
Yes